internal/core/application/blockchain_listener.go

```diff
@@ -86,51 +90,53 @@ func (b *blockchainListener) StopObserveBlockchain() {
 
 func (b *blockchainListener) handleBlockChainEvents() {
 
-	for event := range b.crawlerSvc.GetEventChannel() {
-		e := event.(crawler.AddressEvent)
-		unspents := unspentsFromEvent(e)
-		ctx := context.Background()
-
-		if _, err := b.dbManager.RunUnspentsTransaction(
-			ctx,
-			!readOnlyTx,
-			func(ctx context.Context) (interface{}, error) {
-				return nil, b.updateUnspentsForAddress(ctx, unspents, e.Address)
-			},
-		); err != nil {
-			log.Warnf("trying to update unspents for address %s: %s\n", e.Address, err.Error())
-			break
-		}
+	for crawlerEvent := range b.crawlerSvc.GetEventChannel() {
+		go func(event crawler.Event) {
```

Comment:
let's give this function a name such as `handleEvent` or `eventHandler` or something better, what do you think?

Reply:
Done